copyroles/config.py

```diff
@@ -0,0 +1,38 @@
+import os
+from collections.abc import Iterable
+
+
+class Config:
```

Comment:
Dig it!  I mean... biased... as we use it in other projects.  But hopefully it'll feel handy now and in the future as a place to organize configurations.